README.md

```diff
@@ -118,6 +118,10 @@ To run the project, execute the following command:
 - GitHub: [@zewdu444](https://github.com/zewdu444)
 - Twitter: [@zewdu444](https://twitter.com/zewdu444)
 - LinkedIn: [zewdu-erkyhun](https://www.linkedin.com/in/zewdu-erkyhun-081378b3/)
+
+👤 **Abdimulhin Adan**
+
+- GitHub: [@AbdimulhinYussuf3675](https://github.com/AbdimulhinYussuf3675)
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
```

Comment:
- [ ] OPTIONAL
Kindly ensure that the _Future Features_ section of your ReadMe includes 1-3 potential features to be added later.

Reply:
@rloterh thank you for the feedback we will correct it.